udata/core/reuse/api.py

```diff
@@ -99,8 +99,7 @@ class ReuseListAPI(API):
     @api.expect(Reuse.__index_parser__)
     @api.marshal_with(Reuse.__page_fields__)
     def get(self):
-        query = Reuse.objects(deleted=None, private__ne=True)
-
+        query = Reuse.objects.visible_by_user(current_user)
```

Comment:
Woops, we're changing the behavior since using `visible` that now excludes reuses without datasets.
Is that the reason why visible wasn't used there before?

In any case, I think we want to keep including reuses without datasets here

Reply:
Indeed, and after some investigation, we don't want to return reuses with empty datasets in the `visible()` queryset, as it would break some other places in the code.
I've pushed some partial reverts and rework.